### DISCUSSION TITLE
Slow respond of "* Applying profile" process

### DISCUSSION THREAD

**Discussion by WUIX69** [King-Projects/King-Tweaks#5] [Q&A]
Good Afternoon  @pedrozzz0

Can you fix the slow applying process of profile.. in my case it takes around 2-3min before the "Applying Profile" shows up.. even in startup or in CLI..

Device:
Poco M3

**Currently using KTweaks 2.0.2 Beta 4 (Latest)**

-Thanks Love You 💕

**Comment by pedrozzz0** [comment]
Yep, it may take sometime (about 2 minutes) at boot to run the script, it's normal, i also made it this much cause sometimes scripts aren't updated properly, so this amount of time give sufficient time to the phone connect to the internet after it's native services are started.

**Comment by pedrozzz0** [comment]
Can you send me the logs on internal storage/KTSR also? thanks.

**Reply by WUIX69** [reply]
> Can you send me the logs on internal storage/KTSR also? thanks.

KTSR_DBG.Log
https://ufile.io/oljiqapb

KTSR.Log
https://ufile.io/7r03s8eb

Kxvision.Log
https://ufile.io/johqxjo7
